src/main/java/com/smartbear/readyapi/client/teststeps/soaprequest/SoapRequestStepBuilder.java

```diff
@@ -0,0 +1,79 @@
+package com.smartbear.readyapi.client.teststeps.soaprequest;
+
+import com.smartbear.readyapi.client.assertions.NotSoapFaultAssertionBuilder;
+import com.smartbear.readyapi.client.assertions.SchemaComplianceAssertionBuilder;
+import com.smartbear.readyapi.client.assertions.SoapFaultAssertionBuilder;
+import com.smartbear.readyapi.client.model.Parameter;
+import com.smartbear.readyapi.client.model.SoapRequestTestStep;
+import com.smartbear.readyapi.client.teststeps.TestStepTypes;
+import com.smartbear.readyapi.client.teststeps.request.HttpRequestStepBuilder;
+
+import static com.smartbear.readyapi.client.Validator.validateNotEmpty;
+
+public class SoapRequestStepBuilder extends HttpRequestStepBuilder<SoapRequestStepBuilder, SoapRequestTestStep> {
+
+    public SoapRequestStepBuilder() {
+        super(new SoapRequestTestStep(), TestStepTypes.SOAP_REQUEST.getName());
+    }
+
+    @Override
+    public SoapRequestTestStep build() {
+        super.build();
+
+        validateNotEmpty(getTestStep().getWsdl(), "Missing WSDL location");
+        validateNotEmpty(getTestStep().getBinding(), "Missing WSDL Binding");
+        validateNotEmpty(getTestStep().getOperation(), "Missing WSDL Operation");
+
+        return getTestStep();
+    }
+
+    public SoapRequestStepBuilder withWsdlAt(String wsdlUrl) {
```

Comment:
This method should take a URL parameter instead, to make it clear what it accepts.